apps/docs/src/packages/storybook-utils.md

```diff
@@ -12,11 +12,6 @@ import packageJson from "../../../../packages/storybook-utils/package.json";
 
```

Comment:
now that a link to npm.js exists for this package, do you want to cross-link it here aswell?

Reply:
same goes for the other packages :)

Reply:
Done :)